clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1357,11 +1357,10 @@ of the condition.
 def ConvergentDocs : Documentation {
   let Category = DocCatFunction;
   let Content = [{
-The ``convergent`` attribute can be placed on a function declaration or a
-statement containing call expressions. It is translated into the LLVM
-``convergent`` attribute, which indicates that the call instructions of a
-function with this attribute cannot be made control-dependent on any additional
-values.
+The ``convergent`` attribute can be placed on a function declaration. It is
+translated into the LLVM ``convergent`` attribute, which indicates that the call
+instructions of a function with this attribute cannot be made control-dependent
+on any additional values.
```

Comment:
Maybe should just deprecate this, since it's kind of useless

Reply:
> Maybe should just deprecate this, since it's kind of useless

For languages other that CUDA/HIP, OpenCL, and etc., this attribute may still be helpful to mark functions convergent to restrict code motion to control-equivalent blocks. However, that restriction may not be necessary for those languages.